### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by jgeudens**
Update user manual with more general information about ModbusScope

### CODE REVIEW

File: docs/pages/user_manual.md [hunk 1]
```diff
@@ -1,86 +1,154 @@
...
+![image](../_static/user_manual/overview.png)
 
-## Using
+The most important part of the window is the plot view. The data of the active registers that are polled is shown here as plots. The settings in the axis scale windows can be used to visualize a very specific part of the data or auto scale to show all information. The legend shows an overview of the active register. You can hide a specific register plot by double clicking it in the legend. The same action can be used to show the plot again. When a plot is hidden it is still polled, it is only temporary hidden in the plot view. Markers can be used examine the data in a specific time frame. 
+
+## Installing
 
-*ModbusScope* is a program which can sample Modbus registers and plot the value in a graph to visualize and investigate the dynamic behavior of a system.
+*ModbusScope* can be installed with the provided *.msi* installer. Double-click it and follow the instructions. The installer will install the required files on your computer. At the end of the install process, you can also select to open *.mbs* files with *ModbusScope* by default.
```
Comment (ben-edna) [inline docs/pages/user_manual.md:15]: Given Modbusscope is corss-platform. What are the options for installing on Linux/MacOS?

File: docs/pages/user_manual.md [hunk 2]
```diff
@@ -1,86 +1,154 @@
 # ModbusScope - User Manual
 
-This user manual describes how to get started with *ModbusScope*.
+This user manual describes how to get started with *ModbusScope*. *ModbusScope* is an application which can sample Modbus registers and plot the value in a graph to visualize and investigate the dynamic behavior of a system.
 
-## Installing
+## Overview
```
Comment (ben-edna) [inline docs/pages/user_manual.md:5]: A quick start section could help people get up-and-running quickly. Something along the lines of:

```md
## Quickstart

A quick start to use modbusscope:

* Add a register using the Register Settings
* Press play to start logging
* Press stop 

```

File: docs/pages/user_manual.md [hunk 3]
```diff
@@ -1,86 +1,154 @@
...
+
+The main screen of ModbusScope can contain multiple docked windows. Most are visible at the initial start of the application, except the marker window. This docked windows only appears when a marker has been enabled.
 
-*ModbusScope* can be installed with the provided *.msi* installer. Double-click it and follow the instructions. 
+![image](../_static/user_manual/overview.png)
 
-## Using
+The most important part of the window is the plot view. The data of the active registers that are polled is shown here as plots. The settings in the axis scale windows can be used to visualize a very specific part of the data or auto scale to show all information. The legend shows an overview of the active register. You can hide a specific register plot by double clicking it in the legend. The same action can be used to show the plot again. When a plot is hidden it is still polled, it is only temporary hidden in the plot view. Markers can be used examine the data in a specific time frame.
```
Comment (ben-edna) [inline docs/pages/user_manual.md:11]:
```suggestion
The most important part of the window is the plot view. The data of the active registers that are polled is shown here as plots. The settings in the axis scale windows can be used to visualize a very specific part of the data or auto scale to show all information. The legend shows an overview of the active registers. You can hide a specific register plot by double clicking it in the legend. The same action can be used to show the plot again. When a plot is hidden the register is still polled, it is only temporary hidden in the plot view. Markers can be used examine the data in a specific time frame. 
```

**Review by ben-edna — COMMENTED** [review]
Great job! good documentation is always a big win for a project.

A few small points:
- Did you know that you can store your readthedocs configuration [in a yaml file in your repo](https://docs.readthedocs.io/en/latest/config-file/v2.html)?

- The section titles have either titles of use cases (e.g. Import CSV file, Storing an reusing configuration) or of a screen (e.g. Register Settings, Graph View). It might be more consistent for a user to make it all use cases (i.e. Register Settings --> Configuring Registers). 
afbeelding

File: docs/pages/user_manual.md (hunk 1 above)
Comment (jgeudens) [reply docs/pages/user_manual.md:15]: None, at the moment. I removed all mentions of it being cross platform from the documentation. Maybe I (or someone else) will add an maintainable installer for Linux or MacOS.

File: docs/pages/user_manual.md (hunk 2 above)
Comment (jgeudens) [reply docs/pages/user_manual.md]: Added to doc

**Comment by jgeudens** [comment]
> Great job! good documentation is always a big win for a project.
> 
> A few small points:
> 
>     * Did you know that you can store your readthedocs configuration [in a yaml file in your repo](https://docs.readthedocs.io/en/latest/config-file/v2.html)?
> 
>     * The section titles have either titles of use cases (e.g. Import CSV file, Storing an reusing configuration) or of a screen (e.g. Register Settings, Graph View). It might be more consistent for a user to make it all use cases (i.e. Register Settings --> Configuring Registers).

Thank for the tips. I have restructured the documentation and updated some of the titles.

**Comment by spoorcc** [comment]
Nice job! :+1:

**Pull request merged**